configs/config.go

```diff
@@ -0,0 +1,72 @@
+package configs
+
+import (
+	"fmt"
+	"github.com/caarlos0/env/v6"
+	"log"
+	"os"
+	"runtime"
+)
+
+// Config host application configuration
+// todo: some of these values can be moved to the .env
+type Config struct {
+	AwsProfile        string `env:"AWS_PROFILE"`
+	KubefirstLogPath  string `env:"KUBEFIRST_LOG_PATH" envDefault:"logs"`
+	KubectlVersion    string `env:"KUBECTL_VERSION" envDefault:"v1.20.0"`
+	HomePath          string
+	LocalOs           string
+	LocalArchitecture string
+	KubectlClientPath string
+	KubeConfigPath    string
+
+	TerraformVersion string
+	TerraformPath    string
+
+	HelmClientPath string
+	HelmVersion    string
+
+	DryRun                        bool
+	SkipDeleteRegistryApplication bool
+	DestroyBuckets                bool
+
+	KubefirstVersion string
+	InstallerEmail   string
+
+	SkipGitlabTerraform bool
+	SkipBaseTerraform   bool
+}
+
+func ReadConfig() *Config {
+	config := Config{}
+
+	if err := env.Parse(&config); err != nil {
+		log.Println("something went wrong loading the environment variables")
+		log.Panic(err)
+	}
+
+	var err error
+	config.HomePath, err = os.UserHomeDir()
+	if err != nil {
+		log.Panic(err)
+	}
+
+	config.LocalOs = runtime.GOOS
+	config.LocalArchitecture = runtime.GOARCH
+
+	config.KubectlClientPath = fmt.Sprintf("%s/.kubefirst/tools/kubectl", config.HomePath)
+	config.KubeConfigPath = fmt.Sprintf("%s/.kubefirst/gitops/terraform/base/kubeconfig_kubefirst", config.HomePath)
+	config.TerraformPath = fmt.Sprintf("%s/.kubefirst/tools/terraform", config.HomePath)
+	config.HelmClientPath = fmt.Sprintf("%s/.kubefirst/tools/helm", config.HomePath)
+
+	config.TerraformVersion = "1.0.11"
+
+	// todo adopt latest helmVersion := "v3.9.0"
+	config.HelmVersion = "v3.2.1"
+
+	config.KubefirstVersion = "0.1.1"
+
```

Comment:
This was meant to be in single file, to allow automation to generate this files on builds. 

the goal of `versionConstant.go`. 

It can be in a package or not, it is just that was made simple to allow automation to fill it, been deep on a bigger file, automation may create a mess.

Reply:
We can also automate the release process, not sure yet how we can get the current release version, but this library is a good candidate https://goreleaser.com